x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/transforms/pivot/GroupConfig.java

```diff
@@ -150,6 +153,11 @@ public static GroupConfig fromXContent(final XContentParser parser, boolean leni
 
             ensureExpectedToken(XContentParser.Token.FIELD_NAME, token, parser::getTokenLocation);
             String destinationFieldName = parser.currentName();
+            if (!validAggMatcher.reset(destinationFieldName).matches()) {
```

Comment:
Can we use the negation operator ("!") in ES code?

Reply:
true, its is encourage to use `== false`, I took this from aggregations code, fixing